Tools/WebKitTestRunner/Configurations/WebKitTestRunner.xcconfig

```diff
@@ -45,12 +45,5 @@ INCLUDED_SOURCE_FILE_NAMES = mac/main.mm;
 
 CODE_SIGN_ENTITLEMENTS[sdk=macosx*] = Configurations/WebKitTestRunner.entitlements;
 
-CODE_SIGN_IDENTITY[sdk=macosx*] = $(CODE_SIGN_IDENTITY_$(CONFIGURATION));
```

Comment:
I'm slightly curious about why TestWebKitAPI needs CODE_SIGN_IDENTITY in two xcconfigs, but WebKitTestRunner only needs it in DebugRelease.

Reply:
WebKitTestRunner has these same CODE_SIGN_IDENTITY declarations already: https://github.com/WebKit/WebKit/blob/cb8e510d4f433626e5fe6f9ab6333e46f8b337b6/Tools/WebKitTestRunner/Configurations/DebugRelease.xcconfig#L33-L36

So this change aligns TestWebKitAPI's signing identity behavior with WKTR's.

Reply:
Actually, I was misreading, you were right to point out the discontinuity.

Updated both projects to set ad-hoc codesigning unconditionally outside of DebugRelease. This is other production-buildable tools (DumpRenderTree, MiniBrowser, MobileMiniBrowser) do.